_layouts/home.html

```diff
@@ -54,7 +54,7 @@
             {% assign src = post.img_path | append: '/' | append: src | replace: '//', '/' %}
           {% endunless %}
 
-          {% assign alt = post.image.alt | default: 'Preview Image' | replace: '<', '&lt;' | replace: '>', '&gt;' %}
+         {% assign alt = post.image.alt | xml_escape | default: 'Preview Image' %}
```

Comment:
Be careful with indentation:

```suggestion
          {% assign alt = post.image.alt | xml_escape | default: 'Preview Image' %}
```